frontend-v2/src/components/CoursesList/CoursesList.tsx

```diff
@@ -63,8 +66,28 @@ export default function CoursesList() {
 
   return (
     <>
+      {/* SortDropdown Bar */}
+      <div className="mt-4 w-5/6">
+        <div className="space-y-5 isolate w-full">
+          <div className="flex-1 min-w-[150px] max-w-[200px] xs:min-w-full z-10">
```

Comment:
use fractional measurements instead of hardcoding min / max width e.g. min-w-3/12

Reply:
i tried to convert it to fractional measurements, but didn't work as expected